.changeset/green-apes-explode.md

```diff
@@ -0,0 +1,5 @@
+---
+"@melt-ui/svelte": patch
+---
+
+chore: add eslint rule to prevent imports from lib/builders
```

Comment:
No need to add changeset, as a new release for this won't impact users

Reply:
Yeah I'm an idiot

Reply:
Nah you're fine! Thanks 😄